saleor/graphql/order/mutations/order_fulfill.py

```diff
@@ -105,6 +106,12 @@ class Meta:
         permissions = (OrderPermissions.MANAGE_ORDERS,)
         error_type_class = OrderError
         error_type_field = "order_errors"
+        webhook_events_info = [
+            WebhookEventInfo(
+                type=WebhookEventAsyncType.FULFILLMENT_TRACKING_NUMBER_UPDATED,
+                description="Fulfillment tracking number is updated.",
```

Comment:
From changelog it sounds like this event is optional:

> `orderFulfill` mutation if tracking number is updated.

Would be worth to include this info also here.

Reply:
yes this is optionally triggered, good point